x/accesscontrol/keeper/keeper.go

```diff
@@ -250,6 +250,26 @@ func BuildSelectorOps(accessOps []acltypes.AccessOperationWithSelector, senderBe
 				opWithSelector.Operation.IdentifierTemplate,
 				hexStr,
 			)
+		case acltypes.AccessOperationSelectorType_CONTRACT_REFERENCE:
+			// We use this to import the dependencies from another contract address
+			interContractAddress, err := sdk.AccAddressFromBech32(opWithSelector.Selector)
+			if err != nil {
+				return []acltypes.AccessOperationWithSelector{}, err
+			}
+			// TODO: add a circular dependency check here to ignore if we've already seen this contract/identifier in our reference chain
+			// for now, we will just pass in the same message body, this needs to be changed later though
+			// TODO: build new msgbody for the new contract execute / query msg in later milestone tasks
+			wasmDeps, err := k.GetWasmDependencyMapping(ctx, interContractAddress, contractAddr.String(), msgBody, true)
+
+			if err != nil || !wasmDeps.Enabled {
+				// if we have an error fetching the dependency mapping or the mapping is disabled, we want to use the synchronous mappings instead
+				selectedAccessOps = append(selectedAccessOps, types.SynchronousAccessOpsWithSelector()...)
```

Comment:
i think we can just set `selectedAccessOps = SynchronousAccessOpsWithSelector` in this case?

Reply:
I think we would like to append, since previous access operations with their selectors are also contained here within selectedAccessOps

Reply:
done